README.md

```diff
@@ -24,14 +24,14 @@ module "acm_cert" {
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.13.0 |
-| aws | >= 3.0 |
+| terraform | >= 1.4.0, < 2.0.0 |
+| aws | >= 4.3.5 |
 
 ## Providers
 
 | Name | Version |
 |------|---------|
-| aws | >= 3.0 |
+| aws | 4.67.0 |
```

Comment:
Note: The README shows two different AWS versions (4.3.5 and 4.67.0). I think these are typically the same in our other modules.